src/App.js

```diff
@@ -1,14 +1,71 @@
-import React from 'react';
+import React from "react";
+import TodoList from "./components/TodoComponents/TodoList.js";
+import TodoForm from "./components/TodoComponents/TodoForm.js";
+import styled from "styled-components";
+import { width } from 'window-size';
+
+
+const wrapper = styled.div`
+  height:500px;
+  background: blue;
+  border: 2px solid black;
+`
+
+const dummyData = [
+  {
+    task: "Task One",
+    id: new Date(),
+    completed: false
+  },
+  {
+    task: "Task Two",
+    id: new Date(),
+    completed: false
+  },
+  {
+    task: "Task Three",
+    id: new Date(),
+    completed: false
+  }
+];
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor() {
+    super();
+    this.state = {
+      list: dummyData,
+      inputVal: ""
+    };
+  }
+
+  updateInputVal = event => {
+    const { value } = event.target;
+    this.setState({ inputVal: value });
+  };
+
+  submitNewTask = event => {
+    event.preventDefault();
+
+    this.setState(prevState => ({
+      list: prevState.list.concat({
+        id: new Date(),
+        task: prevState.inputVal,
+        completed: false
+      }),
+      inputVal: ""
+    }));
+  };
+
   render() {
     return (
-      <div>
-        <h2>Welcome to your Todo App!</h2>
-      </div>
+      <wrapper>
```

Comment:
The styled div's name needs to be uppercased to work like mentioned above, You want to do `<Wrapper></Wrapper>`